BankApp/BankApp/Client.cpp

```diff
@@ -0,0 +1,66 @@
+#include "Client.h"
+#include <iostream>
+
+void Client::setSMoney() 
+{
+	sMoney = 0.0; //If no argument was used, set value to 0
+}
+
+void Client::setSMoney(double sMoney)
+{
+	this->sMoney = cMoney;
+}
+
+double Client::getSMoney()
+{
+	return sMoney;
+}
+
+void Client::setCMoney()
+{
+	cMoney = 0.0; //If no argument was used, set value to 0
+}
+
+void Client::setCMoney(double cMoney)
+{
+	this->cMoney = cMoney;
+}
+
+double Client::getCMoney()
+{
+	return cMoney;
+}
+
+/* Accounts will be accounted for numerically, in this case, account 1 will refer to a savings account and account 2 will refer to a checking account.
+Upon further implentation, it might be better to use a switch statement for more than 2 accounts
+*/
+bool Client::withdraw(double money, int account)
+{	
+	string warning = "You do not have enough money in your account";
+	
+	//Check to see that a valid account number was chosen 
+	if (account < 0 || account > 2)
```

Comment:
Wouldn't it make more sense to have: 
if(account ==1) {do account 1 stuff}
else if (account ==2){do account 2 stuff}
else{show warning}

Reply:
we good